.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create an Issue to report a bug
+---
+
+<!-- Make sure we don't have an existing Issue that reports the bug you are seeing (both open and closed). 
+If you do find an existing Issue, re-open or add a comment to that Issue instead of creating a new one. -->
+
+### Description
+
+Briefly describe the bug you are facing.
+
+### Steps to reproduce
+
+Provide steps to replicate.
+
+### Observed result
+
+Please provide command output with `--debug` flag set.
+
+### Expected result
+
+Describe what you expected.
+
+### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)
+
+1. OS:
+2. `sam --version`:
+
```

Comment:
```suggestion
3. Docker version
```

Reply:
Think asking for docker version is also a good idea. Thoughts?

Reply:
Good idea. Do we only care about the version or deeper information?

We can suggest one of two commands here:
```
$ docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
```
or 

```
$docker --version
Docker version 18.06.1-ce, build e68fc7a
```

I am inclined for `docker --version` for now. I don't see a need of the other extra info.

Reply:
yeah ```docker --version``` sounds good to me too.

Reply:
Can you update the suggestion? I want to see how this new github feature works :)